pkg/cmd/roachtest/test_impl.go

```diff
@@ -151,6 +152,7 @@ func (t *testImpl) Cockroach() string {
 		return t.StandardCockroach()
 	}
 	t.randomCockroachOnce.Do(func() {
+		t.l.Printf("GLOBAL SEED: %d", roachtestflags.GlobalSeed)
```

Comment:
Is `ROACHTEST GLOBAL SEED` a better message? (Up to you)

Reply:
I don't think this is the seed we should be logging. It would be more useful to log the actual seed passed to the cockroach process (i.e., what `cockroachRandomSeed()` returns) so that people can copy and paste that when trying to reproduce issues.

Reply:
Yeah you're right. I'm wondering now what the use case of ever setting the global seed would be now. I guess there's still randomization that happens outside of the cockroach process that we'd occasionally like to control?

Reply:
> I'm wondering now what the use case of ever setting the global seed would be though

It's now become a relic from pre-Go 1.20 days, when the seed would have a constant default.

> The [math/rand](https://tip.golang.org/pkg/math/rand/) package now automatically seeds the global random number generator (used by top-level functions like Float64 and Int) with a random value, and the top-level [Seed](https://tip.golang.org/pkg/math/rand/#Seed) function has been deprecated

https://tip.golang.org/doc/go1.20

So I'd say it's probably still fine to continue setting the global default while 23.1 is in support, but we don't need to log it as it's not useful data.

> I guess there's still randomization that happens outside of the cockroach process that we'd occasionally like to control?

Yeah, setting the global RNG would have no effect on cockroach processes, it would just change the random values observed by tests. Unfortunately, a lot of tests (and even roachtest itself) make use of the global RNG.

Reply:
Ahh I see, I think setting global seed affected this call that determines `COCKROACH_RANDOM_SEED`

`seed := rand.Int63()`

Which explains why setting either one was working for me in my testing. `cockroachRandomSeed()` is definitely the way to go though.

Quick sanity run on gceworker shows that it prints it to test.log + correctly reflects the seed passed to `ROACHTEST_ASSERTIONS_ENABLED_SEED`.